lib/checks/lists/dlitem.js

```diff
@@ -1,17 +1,22 @@
-const parent = axe.commons.dom.getComposedParent(node);
-const parentTagName = parent.nodeName.toUpperCase();
+let parent = axe.commons.dom.getComposedParent(node);
+let parentTagName = parent.nodeName.toUpperCase();
+let parentRole = axe.commons.aria.getRole(parent, { noImplicit: true });
+
+if (
+	parentTagName === 'DIV' &&
+	['presentation', 'none', null].includes(parentRole)
+) {
+	parent = axe.commons.dom.getComposedParent(parent);
+	parentTagName = parent.nodeName.toUpperCase();
+	parentRole = axe.commons.aria.getRole(parent, { noImplicit: true });
+}
 
 // Unlike with UL|OL+LI, DT|DD must be in a DL
 if (parentTagName !== 'DL') {
 	return false;
 }
 
-const parentRole = (parent.getAttribute('role') || '').toLowerCase();
-if (!parentRole || !axe.commons.aria.isValidRole(parentRole)) {
```

Comment:
isValidRole is done by `aria.getRole()`, in case you're wondering.

Reply:
I'm surprised it works with the `noImplicit: true`, can you explain that bit? This applies to native elements as well as bolt-on roles, right? The tests seem to pass, so 🤷‍♀️

Reply:
The requirements here are: It needs to be in a `dl`, and if that `dl` has a role attribute, it can only be a `list`. So what's happening is first I check the parentTagName, and then I check if the explicit role (which is what we get from `noImplicit: true` is `null` or `'list'`.